modules/libcom/src/error/errlog.c

```diff
@@ -641,8 +639,8 @@ static void errlogThread(void)
         } else {
             /* snapshot and swap buffers for use while unlocked */
             size_t nLost = pvt.nLost;
-            FILE *console = pvt.toConsole ? pvt.console : NULL;
-            int ttyConsole = pvt.ttyConsole;
+            FILE *console = pvt.toConsole ? pvt.console ? pvt.console : stderr : NULL;
+            int ttyConsole = console ? isATTY(console) : 0;
```

Comment:
Rather than evaluating `stderr` and `isATTY(stderr)` every time, do so once on entry to `errlogThread()` as local variables.  The same for `stdout`.